dgbpy/hdf5.py

```diff
@@ -288,12 +291,12 @@ def getCubeLets( infos, collection, groupnm ):
   img2img = isImg2Img( infos )
   iscluster = isSegmentation( infos )
   examples = infos[exampledictstr]
+  h5file = odhdf5.openFile( infos[filedictstr], 'r' )
   if img2img:
     outnrattribs = 1
   outdtype = np.float32
   if isclass:
-    outdtype = getOutdType(np.array(infos[classesdictstr]))
-  h5file = odhdf5.openFile( infos[filedictstr], 'r' )
+    outdtype = getOutdType(np.array(infos[classesdictstr]), odhdf5.getText(h5file["++info++"], 'Withunlabeled'))
```

Comment:
'Withunlabeled' should be added to dgbpy.keystr.py, in the 'Value keys' section 
Also create there the corresponding dictionary key (lower case convention) in the 'Dictionary key' section.

Read this information in dgbpy.hdf5.getInfo, then you won't have to get the object 'h5file', it is already retrieved at the start of this function.

To read the attribute 'Withunlabeled', because it stored a boolean string, exclusively using odpy.hdf5.getBoolValue( info,  'Withunlabeled' ).
Do it only if the key is present (use odpy.hdf5.hasAttr), and append that boolean to the info dictionary only if the key is present in the file.

Then make a global function dgbpy.hdf5.hasUnlabeled( info ), like the functions dgbpy.hdf5.isSeisClass